docs/split-fasta.md

```diff
@@ -53,11 +53,11 @@ moPepGen splitFasta \
   --output-prefix path/to/split
 ```
 
-This example outputs three split FASTA filese, `split_Coding.fasta`, `split_RNAEditing.fasta`, and `split_Remaining.fasta`.
+This example outputs three split FASTA files, `split_Coding`.fasta`, `split_RNAEditing.fasta`, and `split_Remaining.fasta`.
 
 ### Additional split
 
-Additional split allows you split the records with the source group specified that would otherwise be placed in to the remaining FASTA. See example below.
+Additional split allows you to split the records with the source group specified that would otherwise be placed into the remaining FASTA. See the example below.
```

Comment:
```suggestion
Additional split allows you to split the records with the source group specified that would otherwise be placed along with all other records exceeding `--max-source-groups` into `remaining.fasta`. See the example below.
```